### ISSUE TITLE
Dock unhides on second screen while the primary has a fullscreen game opened

### ISSUE THREAD

**Issue by Lukypie** [micheleg/dash-to-dock#911]
I'm experiencing this while having the dock on both of my two screens with intellihide enabled for fullscreen windows.
My main monitor is on the left, the other one where the issue occurs, on the right.

The dock unhides when I'm into a fulllscreen game, so no cursor is showed and the control is trapped here, and randomly, the right dock would show up with no apparent reason and with no apparent mouse movement. For example, moving the mouse in game to the right section or to the left won't change anything, it will just show up randomly a few times in an hour.
 
But with no other collateral effect than this, for example even if the dock shows up the click are still taken to the game as it should be...

Workaround is to make the dock show only on the left screen (the same where I got the game opened)
Or just to have a program windowed not covering the dock on the right so it will not be hidden

**Comment by micheleg** [comment]
Strange. Is the intellihide functionality working properly (i.e. can you show the dock and click on it by moving on the screen edge)?

**Comment by Lukypie** [comment]
Yes, it is, everything works as it should unless the use case I mentioned 

Basically while inside a fullscreen game some action I make inside it makes the dock unhide itself like I'm hovering the mouse on the other screen dock location

**Comment by Lukypie** [comment]
![ezgif com-video-to](https://user-images.githubusercontent.com/7904559/54568879-38baae00-49d9-11e9-932c-71b857f16f91.gif)

Here is a small gif
